cmd/pj-rehearse/main.go

```diff
@@ -61,8 +63,11 @@ func gatherOptions() options {
 
 	fs.BoolVar(&o.dryRun, "dry-run", true, "Whether to actually submit rehearsal jobs to Prow")
 	fs.BoolVar(&o.noFail, "no-fail", true, "Whether to actually end unsuccessfuly when something breaks")
+	fs.BoolVar(&o.local, "local", false, "Whether this is a local execution or part of a CI job")
 	fs.BoolVar(&o.allowVolumes, "allow-volumes", false, "Allows jobs with extra volumes to be rehearsed")
 
+	fs.StringVar(&o.namespace, "n", "", "Short for --namespace")
```

Comment:
Will anyone ever set this to something other than `ci-stg`? Why expose it to the user?

Reply:
Probably not (we do have some creative users), but I still think it is a nice option to have.

How about we keep the flag but default to `ci-stg` when `o.local`?

Reply:
Creative users cause us headaches. Let's not expose this until we have a clear need, right now I think we simply

```go
namespace := "ci"
if o.runLocal {
    namespace = "ci-stg"
}
```

or whatever